### PR TITLE
Basic code refactoring & readability improvement.

### PR DESCRIPTION
**Pull request by msampathkumar**
Gist of changes
* Fix spell issues
* Simplify Readme page.
* Re-org documentation.
* Code-refactoring scripts.
* Corrected urls to notebooks.
* Added Documentation url at learning part.
* Added missing heading tag for Installation in README.

If its okay, later will be working on code-testing, Python2&Python3 and also any suggestion you may provide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,16 +23,21 @@ import swifter
...
+df['x2'] = df['x'].apply(lambda x: x**2)
+# runs on multiple cores
 df['x2'] = df['x'].swifter.apply(lambda x: x**2)
-df['outCol'] = df['inCol'].swifter.apply(my_func)
-df['outCol'] = df['inCol'].swifter.apply(my_func, positional_arg, keyword_arg=keyword_argval)
 
+# use swifter apply on whole dataframe
 df['agg'] = df.swifter.apply(lambda x: x.sum() - x.min())
+
 df['outCol'] = df[['inCol1'], ['inCol2']].swifter.apply(my_func)
```
Comment (jmcarpenter2) [inline README.md]: df['outCol'] = df[['inCol1', 'inCol2']].swifter.apply(my_func)

File: README.md [hunk 2]
```diff
@@ -21,16 +23,21 @@ import swifter
...
 df['outCol'] = df[['inCol1'], ['inCol2'], ['inCol3']].swifter.apply(my_func, positional_arg, keyword_arg=keyword_argval)
```
Comment (jmcarpenter2) [inline README.md]: df['outCol'] = df[['inCol1', 'inCol2', 'inCol3']].swifter.apply(my_func, positional_arg, keyword_arg=keyword_argval)

File: docs/changelog.md [hunk 3]
```diff
@@ -27,10 +27,10 @@ Fixed bug that would allow certain functions to be applied to the entire series/
 
 ## Version 0.140
 Added support for vectorized and pandas applies to dataframes.
-Converted all string manipulations to pandas apply (unless vectorizable) because dask processes string manipulations slowly.ls
+Converted all string manipulations to pandas apply (unless vectorizable) because dask processes string manipulations slowly.
 
 ## Version 0.13
-Removed numba jit function, because this was adding to the total runtime. Will do some experiments and consider readding later.
+Removed numba jit function, because this was adding to the total runtime. Will do some experiments and consider reading later.
```
Comment (jmcarpenter2) [inline docs/changelog.md]: re-adding, not reading

File: swifter/swifter.py [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
 import timeit
 import warnings
+import pandas as pd
+
+from psutil import cpu_count
 from tqdm import tqdm
-from .tqdm_dask_progressbar import TQDMDaskProgressBar
+from dask import dataframe as dd
+
+import tqdm_dask_progressbar
```
Comment (jmcarpenter2) [inline swifter/swifter.py]: Pretty sure it needs the .tqdm_dask_progressbar for a relative import when launched as a python package on pip

File: swifter/swifter.py [hunk 5]
```diff
@@ -124,6 +128,7 @@ def rolling(self, window, min_periods=None, center=False, win_type=None, on=None
         return Rolling(self._obj, self._npartitions, self._dask_threshold, self._progress_bar, **kwds)
 
     def _wrapped_apply(self, func, axis=0, broadcast=None, raw=False, reduce=None, result_type=None, args=(), **kwds):
+        # TODO: `reduce` is a standard keyword in Python, prefer an alternate name.
```
Comment (jmcarpenter2) [inline swifter/swifter.py]: reduce is a parameter in pandas.DataFrame.apply, so I don't think we can take this out

File: README.md (hunk 1 above)
Comment (jmcarpenter2) [reply README.md]: P.S. I know I wrote this erroneously, not you, but now that I see it I would love for it to be correct :)

**Comment by msampathkumar** [comment]
sure @jmcarpenter2 Will do that

File: README.md (hunk 2 above)
Comment (msampathkumar) [reply README.md]: I will add this but, in later stages can you please add few details like what positional_arg & keyword_arval should look like  using an example.

**Comment by msampathkumar** [comment]
Sorry! I am raising a fresh pull request!

**Pull request closed without merging**